src/components/tx/SignOrExecuteForm/hooks.ts

```diff
@@ -119,17 +142,17 @@ export const useTxActions = (): TxActions => {
       if (isRelayed && safeTx.signatures.size < safe.threshold) {
         if (txId) {
           safeTx = await signRelayedTx(safeTx)
-          tx = await proposeTx(wallet.address, safeTx, txId, origin)
+          tx = await _propose(wallet.address, safeTx, txId, origin)
         } else {
-          tx = await proposeTx(wallet.address, safeTx, txId, origin)
+          tx = await _propose(wallet.address, safeTx, txId, origin)
           safeTx = await signRelayedTx(safeTx)
         }
         txId = tx.txId
       }
 
       // Propose the tx if there's no id yet ("immediate execution")
       if (!txId) {
-        tx = await proposeTx(wallet.address, safeTx, txId, origin)
+        tx = await _propose(wallet.address, safeTx, txId, origin)
         txId = tx.txId
```

Comment:
I think we can not remove it because the `txId` is an optional parameter and might not be defined by the consumer. Looks like we use it only in the SignOrExecuteForm component but there the txId is also optional. We left it as optional because in the wrapper component, both parameters we're passing to it are optional: `txId={props.txId || txDetails?.txId}`